spec/plsql/package_spec.rb

```diff
@@ -35,6 +35,24 @@
     expect(PLSQL::Package.find(plsql, :test_package)).not_to be_nil
   end
 
+  context "with current_schema set" do
```

Comment:
One may say that `current_schema` is always set (by default to the connected user). So it might be good to be a bit more specific in the description here.